packages/suite-data/files/translations/pt.json

```diff
@@ -78,6 +78,7 @@
   "LOG_DESCRIPTION": "Em caso de comunicação com a nossa equipa de apoio, existe um registo contendo todas as informações técnicas necessárias",
   "LOG_INCLUDE_BALANCE_DESCRIPTION": "Caso a sua emissão não esteja relacionada com o seu saldo ou transações, você pode desligar isso. Os descritores da sua conta (XPubs) não serão incluídos no registo copiado.",
   "LOG_INCLUDE_BALANCE_TITLE": "Incluir saldo relacionado",
+  "MAX_FEE": "",
```

Comment:
Empty string https://crowdin.com/translate/trezor-suite/103/en-ptbr?filter=basic&value=0#q=MAX_FEE

Reply:
good catch, removed in crowdin, I believe there is no need to fix it here since Portuguese is not available in Suite yet